artificial-intelligence/deep-learning/index.json

```diff
@@ -35,6 +35,12 @@
             "url": "https://karpathy.ai/zero-to-hero.html",
             "type": "course",
             "level": "everyone"
+        }, 
+        {
+            "title": "AI by hand",
+            "url": "https://aibyhand.substack.com/",
+            "type": "book",
```

Comment:
It's a newsletter, so I think we can either qualify that as a `website` or `article`, wdyt?